Gruntfile.js

```diff
@@ -192,21 +192,21 @@ module.exports = function(grunt) {
      */
     banner:
       '/*!\n' +
-      ' *              ad$$             $$\n' +
-      ' *             d$"               $$\n' +
-      ' *             $$                $$\n' +
-      ' *   ,adPYba,  $$$$$ $b,dPYba,   $$,dPYba,\n' +
-      ' *  aP\'    \'$: $$    $$P\'   \'$a  $$P\'   \'$a\n' +
-      ' *  $(         $$    $$(     )$  $$(     )$\n' +
-      ' *  "b,    ,$: $$    $$b,   ,$"  $$b,   ,$"\n' +
-      ' *   `"Ybd$"\'  $$    $$`YbdP"\'   $$`Ybd$"\'\n' +
-      ' *                   $$\n' +
-      ' *                   $$\n' +
-      ' *                   ""\n' +
+      ' *               ad$$               $$\n' +
+      ' *              d$"                 $$\n' +
+      ' *              $$                  $$\n' +
+      ' *   ,adPYba.   $$$$$  $$.,dPYba.   $$.,dPYba.\n' +
+      ' *  aP′    `$:  $$     $$P′    `$a  $$P′    `$a\n' +
+      ' *  $(          $$     $$(      )$  $$(      )$\n' +
+      ' *  "b.    ,$:  $$     $$b.    ,$"  $$b.    ,$"\n' +
+      ' *   `"Ybd$"′   $$     $$`"YbdP"′   $$`"YbdP"′\n' +
+      ' *                     $$\n' +
+      ' *                     $$\n' +
+      ' *                     $$\n' +
       ' *\n' +
       ' *  <%= pkg.name %> - v<%= pkg.version %>\n' +
-      ' *  <%= pkg.homepage %>' +
-      ' *  A public domain work of the Consumer Financial Protection Bureau\n' +
+      ' *  <%= pkg.homepage %>\n' +
+      ' *  Licensed <%= pkg.license %> by <%= pkg.author.name %> <<%= pkg.author.email %>>\n' +
```

Comment:
Reverted in https://github.com/cfpb/cfgov-refresh/commit/cd4543d6c4da636c8f562be5876cb7699caf84e2